examples/bri-3/src/bri/auth/strategy/local.strategy.ts

```diff
@@ -0,0 +1,16 @@
+import { Strategy } from 'passport-local';
+import { PassportStrategy } from '@nestjs/passport';
+import { Injectable, UnauthorizedException } from '@nestjs/common';
+import { AuthService } from '../services/auth.service';
+
+//use this to check if the user exists in DB, and assign a jwt token if it exists
+@Injectable()
+export class LocalStrategy extends PassportStrategy(Strategy) {
```

Comment:
what is the difference between local and jwt strategies? why do we need both?

Reply:
js doc was already added to the classes.